packages/editor-ui/src/components/SettingsLogStreaming/EventDestinationSettingsModal.ee.vue

```diff
@@ -22,338 +20,336 @@ import {
+	const deleteConfirmed = await confirm(
+		i18n.baseText('settings.log-streaming.destinationDelete.message', {
+			interpolate: { destinationName: destination.label! },
+		}),
+		i18n.baseText('settings.log-streaming.destinationDelete.headline'),
+		{
+			type: 'warning',
+			confirmButtonText: i18n.baseText(
+				'settings.log-streaming.destinationDelete.confirmButtonText',
+			),
+			cancelButtonText: i18n.baseText('settings.log-streaming.destinationDelete.cancelButtonText'),
 		},
-		callEventBus(event: string, data: unknown) {
-			if (this.eventBus) {
-				this.eventBus.emit(event, data);
+	);
+
+	if (deleteConfirmed !== MODAL_CONFIRM) {
+		return;
+	} else {
+		callEventBus('remove', destination.id);
+		uiStore.closeModal(LOG_STREAM_MODAL_KEY);
+		uiStore.stateIsDirty = false;
+	}
+}
+
+function onModalClose() {
+	if (!hasOnceBeenSaved.value) {
+		workflowsStore.removeNode(node.value);
+		if (nodeParameters.value.id && typeof nodeParameters.value.id !== 'object') {
+			logStreamingStore.removeDestination(nodeParameters.value.id.toString());
+		}
+	}
+	ndvStore.activeNodeName = null;
+	callEventBus('closing', destination.id);
+	uiStore.stateIsDirty = false;
+}
+
+async function saveDestination() {
+	if (unchanged.value || !destination.id) {
+		return;
+	}
+	const saveResult = await logStreamingStore.saveDestination(nodeParameters.value);
+	if (saveResult) {
+		hasOnceBeenSaved.value = true;
+		testMessageSent.value = false;
+		unchanged.value = true;
+		callEventBus('destinationWasSaved', destination.id);
+		uiStore.stateIsDirty = false;
+
+		const destinationType = (
+			nodeParameters.value.__type && typeof nodeParameters.value.__type !== 'object'
+				? `${nodeParameters.value.__type}`
+				: 'unknown'
+		)
+			.replace('$$MessageEventBusDestination', '')
+			.toLowerCase();
+
+		const isComplete = () => {
+			if (isTypeWebhook.value) {
+				const webhookDestination = destination as MessageEventBusDestinationWebhookOptions;
+				return webhookDestination.url !== '';
+			} else if (isTypeSentry.value) {
+				const sentryDestination = destination as MessageEventBusDestinationSentryOptions;
+				return sentryDestination.dsn !== '';
+			} else if (isTypeSyslog.value) {
+				const syslogDestination = destination as MessageEventBusDestinationSyslogOptions;
+				return (
+					syslogDestination.host !== '' &&
+					syslogDestination.port !== undefined &&
+					// @ts-expect-error TODO: fix this typing
+					syslogDestination.protocol !== '' &&
+					syslogDestination.facility !== undefined &&
+					syslogDestination.app_name !== ''
+				);
 			}
-		},
-	},
-});
+			return false;
+		};
+
+		useTelemetry().track('User updated log streaming destination', {
+			instance_id: useRootStore().instanceId,
+			destination_type: destinationType,
+			is_complete: isComplete(),
+			is_active: destination.enabled,
+		});
+	}
+}
+
+function callEventBus(event: string, data: unknown) {
+	if (eventBus) {
+		eventBus.emit(event, data);
+	}
+}
```

Comment:
Replace all $locale in the template with i18n

Reply:
I thought I did this already. Thanks for catching this 🙏🏽